src/functions/keycloak/list-users/handler.ts

```diff
@@ -27,22 +15,24 @@ const listUsersHandler: APIGatewayProxyHandler = async () => {
 
     const users = await api.getUsers();
 
-    const mappedUsers: Response[] = users.map((user) => {
-      const responseUser: Response = {
+    const filteredUsers: User[] = [];
+
+    for (let i = 0; i < users.length; i++) {
```

Comment:
This should be done with a map, unless there is a good reason not to?

Reply:
Sure, this was note from Ilmo to avoid mapping. There was something with SonarCloud giving negative feedback about mapping.